### DISCUSSION TITLE
[Bug] Cannot generate a navigation menu using Webstudio AI

### DISCUSSION THREAD

**Discussion by azuredusk10** [webstudio-is/webstudio-community#82] [Q&A]
I entered the prompt:
"Add a navigation menu with a logo on the left and 4 links on the right. On mobile, the links move into a hamburger menu."

Webstudio AI added in several elements to the page, but they don't show up in the main panel. Instead, an error message appears, saying:

```
Error
`NavigationMenuTrigger` must be used within `NavigationMenuItem`

```
![image](https://github.com/webstudio-is/webstudio-community/assets/18142893/9e39f2c8-3aaf-4bb9-a699-b31441b60e51)

**Comment by itsNintu** [comment]
Thanks for the report @azuredusk10 , looks like the AI has broken some of the radix stuff. 

@TrySound can we exclude nav menu from AI? 

@kof FYI

**Reply by kof** [reply]
I am surprised AI used Radix at all, we have not included Radix yet in the list of components AI can use for reasons like this, we need to teach it about the specific structure radix reuiqres
